server.js

```diff
@@ -1,5 +1,11 @@
 const express = require("express");
 const app = express();
+const {OAuth2Client} = require('google-auth-library');
+var fs = require("fs");
+
+const CLIENT_ID="307901483170-m56a98qcu5hjrtknsttovn1niepmdfn2.apps.googleusercontent.com"; // add your own ID here
```

Comment:
When setting up the real prod environment, the customer needs to create a google app in Google Apis and use their own client ID